core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -168,7 +168,7 @@
       display: none;
     }
     .oppia-exploration-footer .oppia-navbar-footer-info-icon {
-      margin-top: -11px;
+      margin-top: -1px;
```

Comment:
Going to be really picky here, but could you make it -3px? I actually think just a tiny bit above center looks a bit nicer so that it seems a bit more aligned with the feedback button, which is also a bit above center.